<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Juegos Matemático</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rbtn_suma">
         <property name="text">
          <string>Suma</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="accessibleName" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtn_resta">
         <property name="text">
          <string>Resta</string>
         </property>
         <property name="accessibleName" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtn_mult">
         <property name="text">
          <string>Multiplicación</string>
         </property>
         <property name="accessibleName" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtn_div">
         <property name="text">
          <string>División</string>
         </property>
         <property name="accessibleName" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ingresa tu resultado:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledit_result">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="accessibleName" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_result">
         <property name="text">
          <string>Resultado</string>
         </property>
         <property name="accessibleName" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>97</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="ledit_1">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="accessibleName" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_op">
         <property name="text">
          <string>+</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="accessibleName" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledit_2">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="accessibleName" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_new">
       <property name="text">
        <string>Nuevo Juego</string>
       </property>
       <property name="accessibleName" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="feedback">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>260</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFelipe_garrido">
   <property name="text">
    <string>Felipe Garrido</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
